### DISCUSSION TITLE
Genomic Resources Working Group Meeting Reminders

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7926] [Announcements]
@ArctosDB/genomic-resources-working-group 

A reminder that the Genomic Resources Working Group has a meeting set for July 12 at 10AM PDT in the Arctos Zoom Room.

Please add topics of interest to the [agenda](https://docs.google.com/document/d/1Qs-RQKkIqpJr5xx2VbfDnxPbw4vFDcypGWcVDjyBYo4/edit#heading=h.l184acat2h1n). If there are no items on the agenda at the time of the meeting I will cancel.

**Comment by happiah-madson** [comment]
Reminder that there is a genomic resources working group meeting today at 1 pm EST (10 am PST). Agenda is here: https://docs.google.com/document/d/1Qs-RQKkIqpJr5xx2VbfDnxPbw4vFDcypGWcVDjyBYo4/edit?tab=t.0#heading=h.uxwcvrnws1wk

Please add any items you may have for discussion, otherwise we will be focusing on working on DwC mapping for GGBN.

**Comment by campmlc** [comment]
Apologies, looks like I won't be able to make it today.

Mariel Campbell
Senior Collection Manager
Division of Genomic Resources
Museum of Southwestern Biology
University of New Mexico
Albuquerque NM 87131
***@***.***
________________________________
